selfdrive/car/volkswagen/pqcan.py

```diff
@@ -23,11 +31,24 @@ def create_lka_hud_control(packer, bus, ldw_stock_values, enabled, steering_pres
   return packer.make_can_msg("LDW_Status", bus, values)
 
 
-def create_acc_buttons_control(packer, bus, gra_stock_values, counter, cancel=False, resume=False):
-  values = gra_stock_values.copy()
+def create_acc_buttons_control(packer, bus, gra_stock_values, cancel=False, resume=False):
+  values = {s: gra_stock_values[s] for s in [
+    "GRA_Hauptschalt",      # ACC button, on/off
+    "GRA_Typ_Hauptschalt",  # ACC button, momentary vs latching
+    "GRA_Kodierinfo",       # ACC button, configuration
+    "GRA_Abbrechen",        # ACC button, cancel
+    "GRA_Neu_Setzen",       # ACC button, set
+    "GRA_Up_lang",          # ACC button, increase or accel, long press
+    "GRA_Down_lang",        # ACC button, decrease or decel, long press
+    "GRA_Up_kurz",          # ACC button, increase or accel, short press
+    "GRA_Down_kurz",        # ACC button, decrease or decel, short press
+    "GRA_Recall",           # ACC button, resume
+    "GRA_Zeitluecke",       # ACC button, time gap adj
+    "GRA_Sender",           # ACC button, CAN message originator
+  ]}
 
   values.update({
-    "COUNTER": counter,
+    "COUNTER": (gra_stock_values["COUNTER"] + 1) % 16,
     "GRA_Abbrechen": cancel,
     "GRA_Recall": resume,
```

Comment:
looks like you can remove these from above unless you want the comments

Reply:
#28246